driver-core/src/main/com/mongodb/client/model/changestream/OperationType.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2017 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.client.model.changestream;
+
+import static java.lang.String.format;
+
+/**
+ * The {@code $changeStream} operation type.
+ *
+ * @since 3.6
+ */
+public enum OperationType {
+
+    /**
+     * The insert operation type
+     */
+    Insert("insert"),
```

Comment:
I'm surprised checkstyle doesn't complain about the lack of all-caps here.  Do we use this style anywhere else?